packages/core-v2/src/entity-model/in-memory-entity-model.ts

```diff
@@ -29,14 +35,16 @@ export class InMemoryEntityModel implements EntityModel {
      */
     public change(updated: Record<string, Entity>, removed: string[]) {
         // Filter removed entities from updated
-        removed = removed.filter(iri => updated[iri] === undefined);
+        removed = removed.filter((iri) => updated[iri] === undefined);
 
-        this.entities = {...this.entities, ...updated};
+        this.entities = { ...this.entities, ...updated };
         for (const removedId of removed) {
             delete this.entities[removedId];
         }
         for (const listener of this.listeners) {
             listener(updated, removed);
         }
     }
-}
\ No newline at end of file
+}
+
+const createId = () => (Math.random() + 1).toString(36).substring(7);
```

Comment:
Mas na tohle nejakou utilitku hezkou? Popr kam by se to dalo vyoperovat, at je to na lepsim miste?